reloc.ml

```diff
@@ -211,11 +212,12 @@ let run_command cmdline cmd =
   in
   (* note: for Cygwin, using bash allow to follow symlinks to find
      gcc... *)
-  if !toolchain = `CYGWIN64 ||
+  if not Sys.win32 || !toolchain = `CYGWIN64 ||
```

Comment:
I don't think it matters, but given that this works if flexlink is on either native Windows or Cygwin, why not instead have `if Sys.unix ||`?